packages/flutter/lib/src/foundation/change_notifier.dart

```diff
@@ -101,7 +101,7 @@ class ChangeNotifier extends Listenable {
   void notifyListeners() {
     assert(_debugAssertNotDisposed);
     if (_listeners != null) {
-      List<VoidCallback> localListeners = new List<VoidCallback>.from(_listeners);
+      final List<VoidCallback> localListeners = new List<VoidCallback>.from(_listeners);
```

Comment:
it's tempting to suggest we should do a copy-on-write during iteration here, since we now control this more closely.

but that's probably a premature optimisation.